shinken/objects/host.py

```diff
@@ -874,6 +881,31 @@ def add_host_chk_dependency(self, h, status, timeperiod, inherits_parent):
     def add_service_link(self, service):
         self.services.append(service)
 
+
+    def __repr__(self):
+        return '<Host host_name=%r name=%r use=%r />' % (
+            getattr(self, 'host_name', None),
+            getattr(self, 'name', None),
+            getattr(self, 'use', None))
+
+    __str__ = __repr__
+
+    def is_excluded_for(self, svc):
+        ''' Check whether this host should have the passed service be "excluded" or "not included".
+
+        An host can define service_includes and/or service_excludes directive to either
+        white-list-only or black-list some services from itself.
+
+        :type svc: shinken.objects.service.Service
+        '''
+
+        sdescr = svc.service_description
+        if not svc.is_tpl() and self.service_includes:
```

Comment:
Is it possible to call this function on non-template only? As geektphe reworked template and separated template for real object.